gravitee-rest-api-standalone/gravitee-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -11,6 +11,13 @@
 # Please have a look to http://docs.gravitee.io/ for more options and fine-grained granularity
 ############################################################################################################
 
+# Specify the URL of the Management API and UI of this instance, mandatory if you want to connect it to Cockpit
+#console:
+#  ui:
+#    url: http://localhost:3000
+#  api:
+#    url: http://localhost:8083
```

Comment:
As we said in the cockpit part of this PR, maybe we should use placeholders instead of "http://localhost:xxx"

Reply:
As discussed with @jhaeyaert the values in `gravitee.yml` are the default ones, so we will go with the localhost